<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppSettingsPage</class>
 <widget class="QWidget" name="AppSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="4,0,4,0,7">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="QcmSettings" stretch="1,1,1,1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="QcmSettingsLabel">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>MCQ Settings :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="nbrOfEntryLinesSettings" stretch="1,1">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="nbrOfEntryLinesLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>MCQ configuration :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="nbrOfEntryLinesDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>1x3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2x3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3x3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4x3</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HardRefreshSetting" stretch="1,1">
       <item>
        <widget class="QLabel" name="HardRefreshLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Hard Refresh Freq :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="HardRefreshDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>Never</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="RandomnessSetting" stretch="1,1">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="RandomnessLabel">
         <property name="text">
          <string>Randomness Type :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="RandomnessDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>True Random</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weighted Random</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="VocabLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="VocabSettings" stretch="1,1,1,1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="VocabSettingsLabel">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Vocabulary Settings :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="RowPerPageLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="RowPerPageLabel">
         <property name="text">
          <string>Row per page :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="RowPerPageComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="VocabFntSizeLayout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="VocabFntSizeLabel">
         <property name="text">
          <string>Font size for entries :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="VocabFntSizeCombox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>4</number>
         </property>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="DisplayLSSetting">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="DisplayLSLabel">
         <property name="text">
          <string>Display LS for :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DisplayLSDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Files and Directories</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Files</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nothing</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="GlobalSettingsLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="GlobalSettings" stretch="1,1,1,1,1,1,1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="GlobalSettingsLabel">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Global Settings :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="BatteryIndicator" stretch="1,1">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="BatteryIndicatorLabel">
         <property name="text">
          <string>Battery format :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="BatteryIndicatorDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Icon and Text</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Icon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Text</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="DateDisplayFormat" stretch="1,1">
       <item>
        <widget class="QLabel" name="DateDisplayFormatLabel">
         <property name="text">
          <string>Date format :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DateDisplayFormatDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>13:37</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1:37 pm</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="KanaSetPageSettingsLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="KanaHardRefreshLabel">
         <property name="text">
          <string>Aggressive Hard Refresh :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="KanaHardRefreshCheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="WifiLayout" stretch="0,0">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="WifiLabel">
         <property name="text">
          <string>Wifi Status :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="WifiCheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="tristate">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ScreenSaverLayout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="ScreenSaverLabel">
         <property name="text">
          <string>Screensaver :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ScreenSaverDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>On (Everywhere)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>On (Exc. Vocab &amp; Note)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="SleepTimerLayout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="SleepTimerLabel">
         <property name="text">
          <string>Go to sleep after :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="SleepTimerDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>5 minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15 minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30 minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Never</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
